clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -239,6 +239,10 @@ Changes in existing checks
   <clang-tidy/checks/bugprone/unchecked-optional-access>` to support
   ``bsl::optional`` and ``bdlb::NullableValue`` from
   <https://github.com/bloomberg/bde>_.
+  Fixed false positives from smart pointer accessors repeated in checking
+  ``has_value`` and accessing ``value``, by caching the locations returned
+  by the accessors. The option `IgnoreSmartPointerDereference` should no
```

Comment:
The part `by caching the locations returned by the accessors` is IMO not really something a user needs wants to know, as it is an implementation detail.